.github/workflows/create_pull_request.yml

```diff
@@ -0,0 +1,21 @@
+name: Create pull request
+
+on:
+  repository_dispatch:
+    types: [create-pr]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Create pull request for ${{ github.event.client_payload.ReleaseBranchName }}
+      uses: repo-sync/pull-request@v2
```

Comment:
I'm surprised there isn't an easier way to create a PR since all the code is already pushed. This Action clones the whole repo locally. I think you can probably create one much easier by using https://github.com/actions/github-script to access https://octokit.github.io/rest.js/v18#pulls

Reply:
I didn't know about this way, it's great, thank you! PR is reworked.